lib/Graphics/Controller.js

```diff
@@ -137,7 +137,10 @@ class GraphicsController extends CoreBase {
 		const variableValue = buttonStyle?.text
 		setImmediate(() => {
 			this.instance.variable.setVariableValues('internal', {
-				[`b_text_${page}_${bank}`]: variableValue,
+				[`b_text_${page}_${bank}`]: {
+					isSingleValue: true,
+					value: variableValue,
+				},
```

Comment:
I am not keen on this object structure it is using, but as some variables are objects (and might remain that way, for use with jsonpath?), I needed a way to differentiate how to treat them.